content/en/docs/tasks/observability/distributed-tracing/lightstep/index.md

```diff
@@ -23,174 +23,377 @@ This task uses the [Bookinfo](/docs/examples/bookinfo/) sample application as an
 
 ## Before you begin
 
-1.  Ensure you have a Lightstep account. [Sign up](https://go.lightstep.com/trial) for a free trial of Lightstep.
+1. Ensure you have a SNCO account. You can use the free community tier and sign up [here](https://info.servicenow.com/developersignup.html).
 
-1.  If you're using [on-premise Satellites](https://docs.lightstep.com/docs/learn-about-satellites#on-premise-satellites), ensure you have a satellite pool configured with TLS certs and a secure GRPC port exposed. See
-    [Install and Configure Satellites](https://docs.lightstep.com/docs/install-and-configure-satellites) for details about setting up satellites.
+1. Ensure sure you have a SNCO [access token](https://docs.lightstep.com/docs/create-and-manage-access-tokens). Access tokens allow your app to communicate with your SNCO project.
 
-    For [Lightstep Public Satellites](https://docs.lightstep.com/docs/learn-about-satellites#public-satellites) or [Developer Satellites](https://docs.lightstep.com/docs/learn-about-satellites#developer-satellites), your satellites are already configured. However you need to download [this certificate](https://docs.lightstep.com/docs/instrument-with-istio-as-your-service-mesh#cacertpem-file) to a local directory.
+## Setup Istio
 
-1.  Ensure sure you have a Lightstep [access token](https://docs.lightstep.com/docs/create-and-manage-access-tokens). Access tokens allow your app to communicate with your Lightstep project.
+Before installing Istio you will need to have a Kubernetes cluster running a supported version of Kubernetes. For Istio v.19 this is Kubernetes versions 1.25, 1.26, 1.27 or 1.28. See here for a list of supported Kubernetes versions if you are installing a different version of Istio.
```

Comment:
Note to add a link here to check a full list of k8s versions.

Reply:
You don't want to have to main this, so just assume that someone has Istio (or link to a setup guide) and give the specific changes for Cloud Observability.